Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM php:8.3-fpm
+
+ARG user
+ARG uid
+ARG laravel_env
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    libzip-dev \
+    zip \
+    unzip \
+    postgresql \
+    postgresql-client \
+    libpq-dev \
+    jpegoptim  \
+    optipng  \
+    pngquant  \
+    gifsicle \
+    libc-client-dev \
+    libkrb5-dev
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
This should be part of previous script, to make sure cache is removed right away, if not, UnionFS will create TWO layers which will make the image big